src/interpreter/index.ts

```diff
@@ -632,23 +641,16 @@ export class Interpreter implements Expr.Visitor<BrsType>, Stmt.Visitor<BrsType>
                     return left.rightShift(right);
                 } else if (isBrsNumber(left) && isBrsNumber(right)) {
                     return this.addError(
-                        new TypeMismatch({
-                            message:
-                                "Type Mismatch. In a bit shift expression the right value must be >= 0 and < 32.",
-                            left: {
-                                type: left,
-                                location: expression.left.location,
-                            },
-                            right: {
-                                type: right,
-                                location: expression.right.location,
-                            },
-                        })
+                        new RuntimeError(
+                            RuntimeErrorCode.BadBitShift,
+                            "",
+                            expression.right.location
+                        )
                     );
                 } else {
                     return this.addError(
                         new TypeMismatch({
-                            message: "Type Mismatch. Attempting to bit shift non-numeric values.",
+                            message: `Operator ">>" can't be applied to`,
```

Comment:
Are all of these error messages worded the same as the messages that come from the device?